docs-v2/content/en/samples/verify/verify.yaml

```diff
@@ -12,13 +12,15 @@ verify:
     name: integration-test-container
     image: integration-test-container
   executionMode:
-    kubernetesCluster: {}
+    kubernetesCluster:
+      overrides: '{ "apiVersion": "v1", "spec": { ... } }'
 - name: metrics-test-container
   container:
     name: metrics-test-container
     image: metrics-test-container
   executionMode:
-    kubernetesCluster: {}
+    kubernetesCluster:
```

Comment:
Can this be commented out w/ some indication the field is optional?

Reply:
See response above.